CONTRIBUTING.md

```diff
@@ -159,22 +144,30 @@ Before creating your pull request, please make sure to review the `tedana` [styl
 
 ### 5. Test your changes
 
-#### Changes to code
+You can run style checks by running the following
 
-For changes to the codebase, we suggest using our development Docker container which will run all the necessary checks and tests to ensure your code is ready to be merged into `tedana`!
-(This does require that you have a local install of [Docker](https://www.docker.com/products/docker-desktop).)
-You can run all the checks with:
+```
+cd TEDANADIR
+flake8 tedana
+```
 
+and unit/integration tests by running the following:
 ```
-docker run --tty --rm -v ${PWD}:/tedana tedana/tedana-dev:latest run_all_tests
+cd TEDANADIR/tedana/tests
```

Comment:
I don't think you need to navigate into the directory to run pytest. It should find them just from the $TEDANADIR  with `pytest tedana`.